recipe-server/client/control/state/approvalRequests/reducers.js

```diff
@@ -0,0 +1,58 @@
+import { fromJS, Map } from 'immutable';
+import { combineReducers } from 'redux';
+
+import {
+  APPROVAL_REQUEST_FETCH,
+  APPROVAL_REQUEST_FETCH_FAILURE,
+  APPROVAL_REQUEST_FETCH_SUCCESS,
+  APPROVAL_REQUEST_RECEIVE,
+  APPROVAL_REQUESTS_FETCH,
+  APPROVAL_REQUESTS_FETCH_FAILURE,
+  APPROVAL_REQUESTS_FETCH_SUCCESS,
+} from '../action-types';
+
+
+function objects(state = Map({}), action) {
```

Comment:
I'm not tied to `objects` it's just what I used on Morgoth and it sort of matches the Django ORM, so it feels familiar to me, but I don't have strong feelings about this.